app/scripts/constants.js

```diff
@@ -190,6 +190,13 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
               prefixes: [
                 "/browse/routes/"
               ]
+            },
+            {
+              label: "Provisioned Services",
+              href: "/browse/provisioned",
+              prefixes: [
+                "/browse/provisioned/"
+              ]
```

Comment:
We should only add this menu item if service catalog is enabled and the user has authority.

You can add an `isValid` item for the tech preview feature check, and there is a `canI` check for Membership below that you can copy.